Move APIs for service pages (#3512)
viamrobotics/docs: docs/appendix/apis/services/motion.md

---
title: "Motion Service API"
linkTitle: "Motion"
weight: 40
type: "docs"

> description: "The motion service API allows you to move a machine based on a SLAM map or GPS coordinates or to move a machine's components form one location to another."

Author: ```suggestion
description: "Give commands to move a machine based on a SLAM map or GPS coordinates or to move a machine's components form one location to another."
```

icon: true
images: ["/icons/components/arm.svg"]
---

> The motion service API allows you to give commands to your [motion service](/services/motion/) for moving a machine based on a SLAM map or GPS coordinates or for moving a machine's components form one location to another.

Reviewer: ```suggestion
The motion service API allows you to give commands to your [motion service](/services/motion/) for moving a machine based on a SLAM map or GPS coordinates or for moving a machine's components from one location to another.
```
